### DISCUSSION TITLE
Support for 3rd party component libraries like PrimeVue and Vuetify

### DISCUSSION THREAD

**Discussion by ZiadJ** [vueform/vueform#102] [Questions] [answered]
Is there any way to use components other than those that come by default with the framework?

**Comment by adamberecz** [comment, marked as answer]
You can add any custom component to Vueform, even PrimeVue or Vuetify components. We have quite an extensive docs on creating elements: https://vueform.com/docs/creating-elements

So technically all PrimeVue and Vuetify inputs could be implemented, but there's no official support for them (at least for now).
